Training/python/DataLoader.py

```diff
@@ -147,6 +147,7 @@ def compile_classes(file_config, file_scaling):
         R.gInterpreter.Declare(config_parse.create_scaling_input(file_scaling,file_config, verbose=False))
         R.gInterpreter.Declare(config_parse.create_settings(file_config, verbose=False))
         R.gInterpreter.Declare('#include "{}"'.format(_LOADPATH))
+        R.gInterpreter.Declare('#include "TauMLTools/Core/src/exception.hpp"')
```

Comment:
- should `exception.h` be included explicitly here?
- if yes, fix the path
- same for `DataLoader_test.py`

Reply:
fixed